packages/jest-haste-map/src/worker.js

```diff
@@ -67,5 +70,17 @@ export async function worker(data: WorkerMessage): Promise<WorkerMetadata> {
     }
   }
 
-  return {dependencies, id, module};
+  // If a SHA-1 is requested on update, compute it.
+  if (data.sha1) {
```

Comment:
Just to confirm: this will be falsy if watchman already provided the sha1? So it's a combination of "sha1 not provided by crawler/watcher" and "sha1 requested in jest-haste-map options"?

Reply:
That is correct. You can see the combination of both parameters [here](https://github.com/facebook/jest/pull/5834/files/7157eec72ce0c503cf6f5ef8f53d29d526a8e8f6#diff-16f69297ff5bdc25a7a58f3ee8139521R440), on the main process.